backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxSparkPlanExecApi.scala

```diff
@@ -837,6 +837,7 @@ class VeloxSparkPlanExecApi extends SparkPlanExecApi {
       Sig[VeloxBloomFilterAggregate](ExpressionNames.BLOOM_FILTER_AGG),
       Sig[TransformKeys](TRANSFORM_KEYS),
       Sig[TransformValues](TRANSFORM_VALUES),
+      Sig[RaiseError](RAISE_ERROR),
```

Comment:
Maybe, better to move it and the above TransformValues/Keys to ExpressionMappings? Since they are Spark functions, not bound to Velox backend.

Reply:
hey @PHILO-HE CH backend not support `raise_error` that why i put it here,  we only make `raise_error` in velox backend - https://github.com/facebookincubator/velox/pull/10110

Reply:
@gaoyangxiaozhu, I feel it may be better to move it to a common place, and just let CH fall back for it, which may require few code change like the code here: https://github.com/apache/incubator-gluten/blob/1c6910e0eca3d911699686c117d68769c8216776/backends-clickhouse/src/main/scala/org/apache/gluten/utils/CHExpressionUtil.scala#L218

Reply:
oh, i got your point! Let me do the change